### PR TITLE
User form template update and fix of email address autocompleted in username field

### LINKED ISSUE #4968: Chrome autofills username field with e-mail address
A lot of users are created with their email address as the username, it's probably because Chrome can autofill the username field with the email address:

image

The signup form has this:

<tr><td>Nom</td><td><input type="text" name="name"  size="80" autocomplete="name" id="name" /></td></tr>
--
  | <tr><td>Adresse e-mail</td><td><input type="text" name="email"  size="80" autocomplete="email" type="email" /></td></tr>
  | <tr><td>Nom d'utilisateur<br/><span class="info">(lettres non accentuées, chiffres et/ou tirets)</span></td><td><input type="text" name="userid"  size="40" autocomplete="username" id="userid" onkeyup="normalize_string_value(this)" /></td></tr>

I'm not sure how to handle that. If we remove autocomplete="username", then the browser won't remember it. If we do put it, we have the risk that the browser autofills with an email address.

### PR DESCRIPTION
**Pull request by stephanegigandet**
Adopt the new template structure discussed in https://github.com/openfoodfacts/openfoodfacts-server/pull/5274

Fixes #4968 in Javascript (alternate option to the server side fix proposed by @hstauf in https://github.com/openfoodfacts/openfoodfacts-server/pull/5169

### CODE REVIEW

**Review by teolemon — APPROVED** [review]
It would be great to mass rename usernames with gmail or hotmail into them, and notify users.

**Comment by stephanegigandet** [comment]
> It would be great to mass rename usernames with gmail or hotmail into them, and notify users.

Good idea. Filed #5285

File: templates/web/pages/user_form/user_form.tt.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-		\$('.tr_teams').show();
+		\$('#teams_section').show();
 	}
 	\$(document).foundation('equalizer', 'reflow');
 });
 
 if (\$('#pro').prop('checked')) {
 	\$('.pro_org_display').show();
-	\$('.tr_teams').hide();
+	\$('#teams_section').hide();
```
Comment (CharlesNepote) [inline templates/web/pages/user_form/user_form.tt.js:14]: Here we have an id: $('#teams_section') and in user_form_page.tt.html we have a class:
```
    .teams_section {
        display: block;
    }
```
I believe one of those is probably wrong, isn't it.

Comment (stephanegigandet) [reply templates/web/pages/user_form/user_form.tt.js:14]: Good catch. The style is not needed anymore, I'll remove it.

Comment (stephanegigandet) [reply templates/web/pages/user_form/user_form.tt.js:14]: fixed by https://github.com/openfoodfacts/openfoodfacts-server/pull/5287

**Pull request merged**